### ISSUE TITLE
getenv before load_dotenv

### ISSUE THREAD

**Issue by hatkidchan** [depressedtimbit/frick-k0nde#2]
You're getting value from environment variables before loading them. Consider changing order of `getenv` and `load_dotenv`. 

Minimal example of issue:
![image](https://user-images.githubusercontent.com/35804000/121262857-75766680-c8bd-11eb-8cbd-fb95b725a727.png)

**Comment by hatkidchan** [comment]
Diff for fixing that issue:
```
diff --git a/main.py b/main.py
index 9b0ec4c..932b27f 100644
--- a/main.py
+++ b/main.py
@@ -7,6 +7,9 @@ from discord.utils import get



+# Loads Predefined Environment Variables
+load_dotenv()
+
 banwords = ['h0nde', 'twitter.com/h0nde'] # Ban-words for Detecting
 log_channel = getenv('log_channel') # Name of channel of logs
 ban_text = 'User {0} (ID: `{1}`) was banned, take a sip 🥤'
@@ -20,9 +23,6 @@ logging.basicConfig(
 )
 log = logging.getLogger('bot')

-# Loads Predefined Environment Variables
-load_dotenv()
-
 # Initialize Bot Class
 intents = discord.Intents.default()
 intents.members = True
 ```

**Comment by hatkidchan** [comment]
Also looks like `log_channel` is not even used, 'cause on https://github.com/depressedtimbit/frick-k0nde/blob/master/main.py#L44 it's hardcoded to `#general`

**Comment by hatkidchan** [comment]
Silently fixed in 1fc26f115a9574f9b71bb4763abf486b2f53dc34
